docs/sources/userguide/dockervolumes.md

```diff
@@ -28,7 +28,12 @@ persistent or shared data:
 - Data volumes can be shared and reused between containers
 - Changes to a data volume are made directly
 - Changes to a data volume will not be included when you update an image
-- Volumes persist until no containers use them
+- Data volumes persist even if the container itself is deleted
+
+Data volumes are designed to persist data, independent of the containers life 
+cycle. Docker will therefore *never* automatically delete volumes if you delete 
```

Comment:
`Docker will *not* automatically delete volumes when you remove a container, .`?

mostly because its `docker rm` :)